metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-indexing-env.xml

```diff
@@ -73,18 +98,31 @@
         <display-name>Indexing Update Column Family</display-name>
     </property>
     <property>
-        <name>indexing_workers</name>
+        <name>ra_indexing_workers</name>
+        <description>Number of Indexing Topology Workers</description>
```

Comment:
With the name and description the same for the random access and batch topologies; can the user distinguish which is which?

Reply:
Actually, I see the display name is different.  Doh.  But here we are using the terms "Elasticsearch" and "HDFS", rather than "Random Access" and "Batch".  We should probably stick with one nomenclature.

Reply:
That sounds good.  Which one would you guys prefer:
* display names in the mpack of "Elasticsearch" and "HDFS"
* variable prefixes of `es` and `hdfs`

I think I'd prefer the first as I really have been trying to set us up to a world where we can make things pluggable (this is the first baby step).

Reply:
I should add, or some other 3rd option that I haven't anticipated. ;)